modules/cpu/outputs.tf

```diff
@@ -0,0 +1,7 @@
+output "cpu_average_percent_id" {
```

Comment:
Seems to me this should just be `cpu_usage_id`. Why did `percent` get introduced?